src/linuxforhealth/csvtofhir/model/csv/basic.py

```diff
@@ -0,0 +1,28 @@
+from typing import List, Optional
+from pydantic import validator
+
+from linuxforhealth.csvtofhir.fhirutils import csv_record_validator as csvrecord
+from linuxforhealth.csvtofhir.fhirutils import fhir_utils
+from linuxforhealth.csvtofhir.fhirutils.fhir_constants import SystemConstants, ValueDataAbsentReason
+from linuxforhealth.csvtofhir.model.csv.base import CsvBaseModel
+from linuxforhealth.csvtofhir.model.csv.base import BaseModel
+
+
+class TokenDetails(BaseModel):
+    token_name: Optional[str]
+    token_value: Optional[str]
+    token_system: Optional[str]
+
+class BasicCsv(CsvBaseModel):
+    identifierTypeSystem: Optional[str]        # system for token identifiers
+    baseSystem: Optional[str]                  # base system
+    identifierNamePrefix: Optional[str]        # prefix for identifier ids
+    tokenList: Optional[List[str]]             # token identifiers
+    patientInternalIdentifier: Optional[str]   # id
+    # datavant
```

Comment:
Remove line 22 and 27 comments, this support should be generic.
Remove 23, 24, 25, 28 - these should go into a general array for identifiers, where the data contract specifies the name, value, system, just like for the tokens

Reply:
Removed the two comments. I will rework other identifiers are you suggested.

Reply:
Using a list now for other indentifiers